docs/reference/config-example.md

```diff
@@ -10,59 +10,83 @@ version: "3.7"
-      ### HTTP
-      kev.workload.liveness-probe-http-port: 8080        # HTTP Liveness probe port. Only used when using an http probe type.
-      kev.workload.liveness-probe-http-path: /status     # HTTP Liveness probe path. Only used when using an http probe type.
-      ### TCP
-      kev.workload.liveness-probe-tcp-port: 8080         # TCP Liveness probe port. Only used when using an tcp probe type.
-      ### Common to all liveness probe types
-      kev.workload.liveness-probe-initial-delay: 1m0s    # Default: 1m. How long to wait before initial probe run. Shared by all liveness probe types.
-      kev.workload.liveness-probe-interval: 1m0s         # Default: 1m. Interval for the probe. Shared by all liveness probe types.
-      kev.workload.liveness-probe-retries: "3"           # Default: 3. Number of probe retires. Shared by all liveness probe types.
-      kev.workload.liveness-probe-timeout: 10s           # Default: 10s. Probe timeout. Shared by all liveness probe types.
-
-      ## Readiness probe
-      kev.workload.liveness-probe-type: exec          # Default: none. Possible options: none | exec | http | tcp.
-      ### Exec
-      kev.workload.readiness-probe-command: "echo 'n/a'"  # readiness probe command to run. Only used when using exec probe type.
-      ### HTTP
-      kev.workload.readiness-probe-http-port: 8080        # HTTP Readiness probe port. Only used when using an http probe type.
-      kev.workload.readiness-probe-http-path: /status     # HTTP Readiness probe path. Only used when using an http probe type.
-      ### TCP
-      kev.workload.readiness-probe-tcp-port: 8080         # TCP readiness probe port. Only used when using an tcp probe type.
-      ### Common to all readiness probe types
-      kev.workload.readiness-probe-initial-delay: 1m0s    # Default: 1m. How long to wait before initial probe run. Shared by all readiness probe types.
-      kev.workload.readiness-probe-interval: 1m0s         # Default: 1m. Interval for the probe. Shared by all readiness probe types.
-      kev.workload.readiness-probe-retries: "3"           # Default: 3. Number of probe retires. Shared by all readiness probe types.
-      kev.workload.readiness-probe-timeout: 10s           # Default: 10s. Probe timeout. Shared by all readiness probe types.
+    x-k8s:                                               # compose K8s configuration extension
+      disabled: false                                    # When disabled the compose service won't be included in generated K8s manifests. Defaults to 'false'. 
+      service:                                           # K8s service configuration (only required if values are overridden)
+        type: None                                       # Default: none (no service). Possible options: none | headless | clusterip | nodeport | loadbalancer.
+        nodeport:                                        # Default: nil. Only taken into account when working with service.type: nodeport
```

Comment:
It's good. Perhaps we should give an example in the comment? e.g. `Set with numeric value e.g. 5555` or something like that?